PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,8 @@
+<!--
+Please make sure you've read and understood our contributing guidelines;
```

Comment:
Don't we need to remove this file?

Reply:
Why would we need to remove it?

Reply:
The way I understand it, if we have a .github/PULL_REQUEST_TEMPLATE folder, this file won't be used anymore. I would have to test on my fork to be sure :P Feel free to ignore this message if I am mistaken.

Reply:
I think you are mixing PULL_REQUEST_TEMPLATE.md and CONTRIBUTING.md. 

Also CONTRIBUTING.md will still be relevant. It will be shown as "your PR doesn't fit those templates? Open a normal PR instead" kind of like our issues template work. At least that's the desired outcome. Feel free to test on your fork if you need to be sure.